configuration/configuration.go

```diff
@@ -62,7 +65,7 @@ func (c Configuration) HasCustomCommitMessage() bool {
 }
 
 func (c Configuration) IsWipCommitMessage(line string) bool {
-	return strings.HasPrefix(line, c.WipCommitMessage)
+	return strings.HasPrefix(line, c.WipCommitMessage) || strings.HasPrefix(line, StartCISkipCommitMessage)
```

Comment:
Do we have a test fail if that `|| strings.HasPrefix(line, StartCISkipCommitMessage)` is ommited?

Reply:
Didn't. `IsWipCommitMessage` is used only in `squashWipGitEditor`, so it does not need to change. I will restore.